src/views/activityBar/changes/changeTreeView/fileTreeView.ts

```diff
@@ -123,12 +123,17 @@ export class FileTreeView implements TreeItemWithoutChildren {
 			newContent,
 			oldContent,
 		});
+
+		const tabTitle = `${file.filePath} ${
+			patchsetBase?.number ?? 'Base'
+		} -> ${file.currentRevision.number}`;
+
 		return {
 			command: 'vscode.diff',
 			arguments: [
 				oldURI,
 				newURI,
-				path.basename(file.filePath),
+				path.basename(tabTitle),
```

Comment:
This won't work, `path.basename` basically takes a full file path and returns the name of just the file (no folders and stuff). So `a/b/c/d.ts` -> `d.ts`. Instead you want to move this `path.basename` into `tabTitle` around `file.filePath`. :)